mbed_build/_internal/mbed_tools/export.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Export top-level CMakeLists.txt command."""
+import pathlib
+
+import click
+
+from mbed_build.mbed_build import generate_cmakelists_file, write_cmakelists_file
+
+
+@click.command()
+@click.option(
+    "-o",
+    "--output_directory",
+    type=click.Path(),
+    required=True,
+    help="Destination for exported top-level CMakeLists.txt file.",
+)
+@click.option(
+    "-t",
+    "--toolchain",
+    type=click.Choice(["ARM5", "ARM6", "GCC", "IAR"]),
```

Comment:
ARM C5 will not be supported and IAR is still being discussed but best not to put too much effort into this one.

Reply:
Thats interesting, so the only two options will be "ARM6" and "GCC"?

Reply:
If its decided not to continue with IAR

Reply:
e1fbf9b - I have only included the ones we know we will support for now. We can add IAR back in if its wanted later.